core/store/src/trie/mem/construction.rs

```diff
@@ -1,7 +1,9 @@
 use super::arena::Arena;
+use super::freelist::{ReusableVecU8, VecU8Freelist};
 use super::node::MemTrieNodeId;
 use crate::trie::mem::node::InputMemTrieNode;
 use crate::NibbleSlice;
+use elastic_array::ElasticArray4;
```

Comment:
This looks like an iffy crate to use compared to the much better established `smallvec`: the documentation is pretty light, the maintenance status is unclear and its just plain a less popular crate.

Though I guess this is a preexisting dependency, so I'll file an issue to address this instead...